routes/index.js

```diff
@@ -15,8 +18,45 @@ router.get('/badgein', function(req, res) {
   res.render('badgein.njk');
 });
 
-router.get('/registration', function(req, res) {
+router.post('/badgein', function(req, res) {
```

Comment:
Would you be able to remove this post request for badgein since Bishoy is doing it? Just so there is no confusion later on and rebasing issues.

Reply:
Whoops! Definitely. I was just mocking up a proof of concept when I was playing with the database. I didn't realize I checked that in. I'll update, rebase, squash, and re-commit right now.